blueprints/getting-started/main.tf

```diff
@@ -83,12 +81,48 @@ module "eks_blueprint_addons" {
 
   # HashiCorp Vault
   enable_vault = true
-
   vault_helm_config = {
-    namespace = var.namespace
+    namespace = var.vault_namespace
+  }
+
+  tags = local.tags
+}
+
+################################################################################
+# Supporting Resources
+################################################################################
+
+# See https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 3.0"
+
+  name = local.name
+  cidr = var.vpc_cidr
+
+  azs             = local.azs
+  public_subnets  = [for k, v in local.azs : cidrsubnet(var.vpc_cidr, 8, k)]
+  private_subnets = [for k, v in local.azs : cidrsubnet(var.vpc_cidr, 8, k + 10)]
+
+  enable_nat_gateway   = true
+  single_nat_gateway   = true
+  enable_dns_hostnames = true
+
+  # Manage so we can name
+  manage_default_network_acl    = true
+  default_network_acl_tags      = { Name = "${local.name}-default" }
+  manage_default_route_table    = true
+  default_route_table_tags      = { Name = "${local.name}-default" }
+  manage_default_security_group = true
+  default_security_group_tags   = { Name = "${local.name}-default" }
+
+  public_subnet_tags = {
+    "kubernetes.io/role/elb" = 1
+  }
+
+  private_subnet_tags = {
+    "kubernetes.io/role/internal-elb" = 1
   }
 
-  depends_on = [
```

Comment:
We shouldn't depend on other modules and instead map the dependency ordering by passing outputs to the modules inputs for implicit dependency mapping. 

Using `depends_on` here will cause disruptions for users, specifically around IAM policies which cause service disruption - ref https://github.com/hashicorp/terraform/issues/30340#issuecomment-1010202582